From 7de7d6c1c5f37c8a657de5534ace5c677aa3a0a6 Mon Sep 17 00:00:00 2001
From: Emmanuel Gil Peyrot <linkmauve@linkmauve.fr>
Date: Wed, 18 Nov 2020 21:28:29 +0100
Subject: [PATCH 054/194] GTK: Use correct icon name

a8e581dcf328c020c2b7d42229b2f3f18d3dee0e renamed it, but it never got
updated in the UI.
---
 desmume/src/frontend/posix/gtk/main.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/desmume/src/frontend/posix/gtk/main.cpp b/desmume/src/frontend/posix/gtk/main.cpp
index d8123a01..d7b9eb9c 100644
--- a/desmume/src/frontend/posix/gtk/main.cpp
+++ b/desmume/src/frontend/posix/gtk/main.cpp
@@ -1541,7 +1541,7 @@ static void About(GSimpleAction *action, GVariant *parameter, gpointer user_data
             "program-name", "DeSmuME",
             "version", EMU_DESMUME_VERSION_STRING() + 1, // skip space
             "website", "http://desmume.org",
-            "logo-icon-name", "DeSmuME",
+            "logo-icon-name", "org.desmume.DeSmuME",
             "comments", "Nintendo DS emulator based on work by Yopyop",
             "authors", authors,
             NULL);
-- 
2.31.1

